src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -244,6 +255,38 @@
      */
     private       DialogDoneCancel confirmDeleteDialog;
 
+    /**
+     * Check if the tool is in the static schematic mode.
+     */
+    private boolean staticSchematicMode = false;
+
+    /**
+     * Name of the static schematic if existent.
+     */
+    private String staticSchematicName = "";
+
+    /**
+     * Possible free to place structure.
+     */
+    private FreeMode freeMode;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "freeMode" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1068)